biosppy/features/frequency.py

```diff
@@ -0,0 +1,237 @@
+import numpy as np
+from .. import utils
+from . import time
+from scipy import interpolate
+
+
+def getbands(frequencies, fband= [0, 1]):
+   band = np.argwhere((frequencies >= fband[0]) & (frequencies <= fband[1])).reshape(-1)
+   return frequencies[band]
+
+
+def freq_features(signal, SR):
```

Comment:
Use only lowercase variables and change variable name to match standard use.

```suggestion
def freq_features(signal, sampling_rate):
```